### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Flunavgt**
For this PR complete tasks:
Complete Contact form
Created and linked formspree
Set max input char values on contact form
Use flexbox on Form

### CODE REVIEW

File: main.html [hunk 1]
```diff
@@ -144,5 +144,22 @@ <h3 class="worktitle">My Recent Work</h3>
...
+            <input type="email" class="contactemail" placeholder="Email" name="email" required>
+            <textarea name="request" id="request1" cols="30" rows="10" maxlength="500" placeholder="Enter Text here"></textarea>
+        <button type="submit" class="submit">Submit Request</button>
+        </form>
+        <img class="separator" src="./pictures/vector4.png" alt="line to separate information">
+        <div class="findMe">
+            <a href="https://www.linkedin.com/"><img class="link" src="pictures/Linkedin.png" alt=""></a>
+            <a href="https://github.com/Flunavgt"><img class="link" src="pictures/github.png" alt=""></a>
+            <a href="https://twitter.com/?lang=en"><img class="link" src="pictures/twitter.png" alt=""></a>
+        </div>
```
Comment (sumairq) [inline main.html]: - [ ] Lets make the social icons as close to the Figma design as possible . Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
image  |  image

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @Flunavgt , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️professional README
✔️Correct Github flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.html [hunk 2]
```diff
@@ -144,5 +144,22 @@ <h3 class="worktitle">My Recent Work</h3>
...
+            <input type="email" class="contactemail" placeholder="Email" name="email" required>
+            <textarea name="request" id="request1" cols="30" rows="10" maxlength="500" placeholder="Enter Text here"></textarea>
+        <button type="submit" class="submit">Submit Request</button>
+        </form>
+        <img class="separator" src="./pictures/vector4.png" alt="line to separate information">
+        <div class="findMe">
+            <a href="https://www.linkedin.com/"><img class="link" src="pictures/TimeAndIcons/linkedin.png" alt=""></a>
+            <a href="https://github.com/Flunavgt"><img class="link" src="pictures/TimeAndIcons/github.png" alt=""></a>
+            <a href="https://twitter.com/?lang=en"><img class="link" src="pictures/TimeAndIcons/twitter.png" alt=""></a>
+        </div>
```
Comment (tufoinnkuo10) [inline main.html]: - [ ] like the previous reviewer noted, kindly add your social icons so they look like the figma version. they will serve as links to your online sites so they are useful. 
your version 
Screenshot (210)_LI
figma version 
Screenshot (209)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Flunavgt 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ followed figma design 
✅ form submits successfully

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by sumairq — APPROVED** [review]
### Hi @Flunavgt  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Followed Figma template
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**